README.md

```diff
@@ -32,24 +32,19 @@
 
 1. **AAD Application / Service principal**.  The deployed OpenShift cluster
    needs a valid AAD application and service principal to call back into the
-   Azure API, and optionally in order to enable AAD authentication.  There are a
-   few options here:
+   Azure API, and optionally in order to enable AAD authentication. 
+   Application can be created using script `./hack/aad.sh app-create`.
 
-   1. (Ask your Azure subscription administrator to) precreate a generic AAD
```

Comment:
I don't think the doc changes are right, and the updated doc is not very clear.
A SP is always needed; if using AAD, our workflow is to then have a second AAD enabled SP.
So this paragraph should stay, then there should be the (optional) AAD piece below?
Please read through the documentation from the perspective of a MSFT new joiner and check it would make sense to them.